internal/infrastructure/db/migrations/20221006130646_integration_features.sql

```diff
@@ -21,7 +21,22 @@ INSERT INTO integration_features (feature_key, display_name, elastic_property) V
     ('odometer', 'Odometer', 'odometer'),
     ('oil', 'Engine Oil Life', 'oil'),
     ('tires', 'Tires', 'tires.frontLeft'),
-    ('speed', 'Speed', 'speed');
+    ('speed', 'Speed', 'speed'),
+    ('location', 'Location', 'latitude'),
+    ('battery_capacity', 'Battery Capacity', 'batteryCapacity'),
+    ('charging', 'Charging Status', 'charging'),
+    ('range', 'Range', 'range'),
+    ('vin', 'VIN', 'vin'),
+    ('cell_tower', 'Cell Tower Info', 'cell.ip'),
+    ('engine_runtime', 'Engine Run Time', 'runTime'),
+    ('ambient_temperature', 'Ambient Temperature', 'ambientTemp'),
+    ('barometric_pressure', 'Barometric Pressure', 'barometricPressure'),
+    ('coolant_temperature', 'Coolant Temperature', 'coolantTemp'),
+    ('engine_load', 'Engine Load', 'engineLoad'),
+    ('engine_speed', 'Engine Speed', 'engineSpeed'),
+    ('throttle_position', 'Gas Pedal Position', 'throttlePosition'),
```

Comment:
Is throttle the same as gas pedal? Serious question, I don't know much about cars.

Reply:
Lol.. I don't know much either, I read it online that it is 

<img width="638" alt="Screen Shot 2022-10-18 at 11 51 28 AM" src="https://user-images.githubusercontent.com/111298305/196480694-5380c1ab-5672-44e4-b6bb-6a4d732dfb1d.png">